Artifacts/windows-teamviewer/artifactfile.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2015-01-01/dtlArtifacts.json",
+  "title": "Teamviewer",
```

Comment:
Looks like the V should be capitalized, based on their website at the entry in the chocolately. Let's make this TeamViewer.